### ISSUE TITLE
Missing VST

### ISSUE THREAD

**Issue by UlikGames** [musescore/MuseScore#24319]
### Issue type

VST bug

### Description with steps to reproduce

1. Download Grand Rhapsody Piano VST
2. Go VST tab and nothing there

### Supporting files, videos and screenshots

This screenshot from Musescore 4, as you can see, there is a missing Waves tab
![image](https://github.com/user-attachments/assets/7c9c0ae4-8d1f-403c-9610-c8d00adda2dc)
And this screenshots from Dorico 5
![image](https://github.com/user-attachments/assets/675035af-a76f-4df6-ac22-58d5a4a446b3)
![image](https://github.com/user-attachments/assets/9c2bf923-163d-42eb-b192-f29dfb451709)

### What is the latest version of MuseScore Studio where this issue is present?

OS: Windows 10 Version 2009 or later, Arch.: x86_64, MuseScore Studio version (64-bit): 4.4.0-242390800, revision: github-musescore-musescore-0fcd11b

### Regression

I was unable to check

### Operating system

Windows 10

### Additional context

I try to add a custom folder Edit->Preferences->Folders to check because I think maybe it can fix it. No, still can't find it. And other apps can find VST.

### Checklist

- [X] This report follows the [guidelines](https://github.com/musescore/MuseScore/wiki/Reporting-bugs-and-issues) for reporting bugs and issues
- [X] I have verified that this issue has not been logged before, by searching the [issue tracker](https://github.com/musescore/MuseScore/issues) for similar issues
- [X] I have attached all requested files and information to this report
- [X] I have attempted to identify the root problem as concisely as possible, and have used minimal reproducible examples where possible

**Comment by MarcSabatella** [comment]
Did you restart MuseScore Studio after installing the VST?  And did you verify you installed a VST3 version and not an older unsupported format?

**Comment by UlikGames** [comment]
Yes, I restarted. And yes, I download all of them in every format: VST, VST3, RTAS, AAX. But I can reinstall again.
